pkg/crc/validation/validation.go

```diff
@@ -42,6 +42,14 @@ func ValidateDiskSize(value int) error {
 	return nil
 }
 
+func ValidatePersistentVolumeSize(value int) error {
+	if value < constants.DefaultPersistentVolumeSize {
+		return fmt.Errorf("requires disk size in GiB >= %d", constants.DefaultDiskSize)
```

Comment:
this value should be larger than `DefaultPersistentVolumeSize` (15gb) or `DefaultDiskSize` (31gb) ?

Reply:
Ah, good catch. Fixed 👍🏼